<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newuser</class>
 <widget class="QDialog" name="newuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRank">
     <property name="text">
      <string>Выберите должность нового пользователя:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Читатель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Библиотекарь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Администратор</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Введите Имя:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditLastName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelFirstName">
     <property name="text">
      <string>Введите Фамилию</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditFirstName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelPatronymic">
     <property name="text">
      <string>Введите Отчество</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPatronymic"/>
   </item>
   <item>
    <widget class="QLabel" name="labelReaderNumber">
     <property name="text">
      <string>Номер читательского билета:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditReadersNumber"/>
   </item>
   <item>
    <widget class="QLabel" name="labelHome">
     <property name="text">
      <string>Домашний адрес:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditHome"/>
   </item>
   <item>
    <widget class="QLabel" name="labelLogin">
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditLogin"/>
   </item>
   <item>
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPassword"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newuser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newuser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
